### ISSUE TITLE
pin a thread in a column

### ISSUE THREAD

**Issue by theZacAttacks** [tootsuite/mastodon#9920]
### Pitch

Allow users to pin a thread to a column and allow streaming responses to be displayed as they come in.
Maybe also have a streaming endpoint in the API for threads.

### Motivation

Users often want to keep track of threads as they grow and I feel like this is a good way to make it easier to watch a thread without leaving the UI or having to refresh.

**Comment by tommyschnabel** [comment]
+1 for this please

**Comment by tommyschnabel** [comment]
+1, this would be great

**Comment by naught101** [comment]
Yes please! So useful for checking out users involved in a thread.
